test/views/behaviours/ViewBehaviours.scala

```diff
@@ -88,10 +88,10 @@ trait ViewBehaviours extends ViewSpecBase {
           assertEqualsMessage(doc, "title", s"$messageKeyPrefix.title")
         }
 
-        "display the correct page title with caption" in {
+        "display the correct page title with section" in {
 
           val doc = asDocument(view)
-          assertPageTitleWithCaptionEqualsMessages(doc, s"$messageKeyPrefix.caption",  captionParam, s"$messageKeyPrefix.heading")
+          assertPageTitleWithSectionSubheading(doc, s"$messageKeyPrefix", captionParam = "")
```

Comment:
messageKeyPrefix is just a String so

s"$messageKeyPrefix" -> messageKeyPrefix

Reply:
Done